.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
+npm run lint:yaml
```

Comment:
I would expect `npm run lint` to call the `lint:yaml` script. Can we get this fixed on this PR? I checked out CI pipeline and we apparently aren't even linting in CI (🙈) This needs updated at some point.

Reply:
These are two different scripts and I don't want to change the structure in this PR, which is only about updating the Husky version. I'm going to defer that change.

```
"lint": "eslint --fix cypress app/assets/js/scripts.js"
"lint:yaml": "yamllint '*.yml' 'basic/*.yml' .buildkite/pipeline.yml .semaphore/semaphore.yml '.github/workflows/*.yml'"
```

If linting is added to CI then it would make it cleaner to have just one call to do all linting. Other aspects are to check whether all files which require linting are actually being linted.

Also when updating to `eslint-plugin-mocha@latest` the examples fail and it doesn't like the arrow function either.

So there are still quite a few linting topics to cover, and I would plan several follow-on PRs to cover these, including your suggestion to have a single linting script command.